packages/js/components/src/collapsible-content/collapsible-content.tsx

```diff
@@ -33,27 +34,46 @@ export const CollapsibleContent: React.FC< CollapsedProps > = ( {
 		return persistRender ? 'visually-hidden' : 'hidden';
 	};
 
+	const collapsibleToggleId = useInstanceId(
+		CollapsibleContent,
+		'collapsible_toggle'
```

Comment:
The pattern we've been using has been to use `${classname}-${instanceId}`.  In this case:

```suggestion
		'woocommerce-collapsible-content__toggle'
```

But open to hear arguments against this if you have a reason.

Reply:
done -> https://github.com/woocommerce/woocommerce/pull/37760/commits/3f249f52f3271e0b4d362b6788109d2ea92b9249